dbt/adapters/databricks/impl.py

```diff
@@ -716,12 +716,12 @@ def get_from_relation(
 
         relation_config = super(DeltaLiveTableAPIBase, cls).get_from_relation(adapter, relation)
         connection = cast(DatabricksDBTConnection, adapter.connections.get_thread_connection())
-        wrapper: DatabricksSQLConnectionWrapper = connection.handle
-
-        # Ensure any current refreshes are completed before returning the relation config
-        tblproperties = cast(TblPropertiesConfig, relation_config.config["tblproperties"])
-        if tblproperties.pipeline_id:
-            wrapper.cursor().poll_refresh_pipeline(tblproperties.pipeline_id)
+        if relation.type != DatabricksRelationType.StreamingTable:
```

Comment:
How does this fix the issue?  This code was added so that if you were currently running a pipeline, we would wait until that pipeline was complete before doing the dbt run.  I don't see how this could fix the issue described.